PSSwagger/Paths.psm1

```diff
@@ -601,6 +608,88 @@ function New-SwaggerPath
+            }
+            if (-not $serverSideFunctionDetails) {
+                # Warning: Couldn't find specified server-side command
+                Write-Warning "Couldn't find server-side result operation: $($clientSideFilter.ServerSideResultCommand)"
+            }
+            else {
+                $serverSideParameterSet = $serverSideFunctionDetails['ParameterSetDetails'] | Where-Object { $_.OperationId -eq $clientSideFilter.ServerSideResultParameterSet}
+                if (-not $serverSideParameterSet) {
+                    # Warning: Couldn't find specified server-side parameter set
+                    Write-Warning "Couldn't find server-side result parameter set: $($clientSideFilter.ServerSideResultParameterSet)"
+                }
+                else {
+                    $clientSideParameterSet = $parameterSetDetails | Where-Object { $_.OperationId -eq $clientSideFilter.ClientSideParameterSet }
+                    if (-not $clientSideParameterSet) {
+                        Write-Warning "Couldn't find client-side parameter set: $($clientSideFilter.ClientSideParameterSet)"
+                    }
+                    else {
+                        $valid = $true
+                        foreach ($parametersDetail in $serverSideParameterSet.ParameterDetails) {
+                            foreach ($parameterDetailEntry in $parametersDetail.GetEnumerator()) {
+                                if ($parameterDetailEntry.Value.Mandatory -eq '$true' -and
+                                ((-not $parameterDetailEntry.Value.ContainsKey('ReadOnlyGlobalParameter')) -or $parameterDetailEntry.Value.ReadOnlyGlobalParameter) -and
+                                ((-not $parameterDetailEntry.Value.ContainsKey('ConstantValue')) -or $parameterDetailEntry.Value.ConstantValue)) {
+                                    $clientSideParameter = $null
+                                    foreach ($pd in $clientSideParameterSet.ParameterDetails.GetEnumerator()) {
+                                        foreach ($entry in $pd.GetEnumerator()) {
+                                            if (($entry.Value.Mandatory -eq '$true') -and ($entry.Value.Name -eq $parameterDetailEntry.Value.Name)) {
+                                                $clientSideParameter = $entry
+                                            }
+                                        }
+                                    }
+                                    if (-not $clientSideParameterSet) {
+                                        # Warning: Missing client-side parameter
+                                        Write-Warning "Required server-side parameter '$($parameterDetailEntry.Value.Name)' is not required by the client-side, which will cause issues in client-side filtering. Can't include client-side filtering."
+                                        $valid = $false
```

Comment:
>$valid = $false [](start = 40, length = 15)

It looks like this variable is assigned but not used. Please consider removing if it is not required.